test/ContractGenerator.Tests/ProtoUtilsTests.cs

```diff
@@ -67,6 +67,61 @@ public void GetClassName_ReturnsCorrectClassName()
         Assert.Equal("global::AElf.Contracts.HelloWorld.HelloWorld", className);
     }
 
+    [Fact]
+    public void GetCsharpComments_ReturnsComments()
+    {
+        // Arrange: Create a DescriptorBase with a known FullName and File
+        var fds = GetFileDescriptorSet("helloworld");
+        var byteStrings = fds.File.Select(f => f.ToByteString());
+        var fileDescriptors = FileDescriptor.BuildFromByteStrings(byteStrings, _extensionRegistry);
+        var file = fileDescriptors[^1];
+
+        // Act: Call the GetClassName method
+        var comments = ProtoUtils.GetCsharpComments(file, true);
+        const string expectedComments = @"//
```

Comment:
This expected output doesn't seem right. We need to fix either the code or the test case.

Reply:
yeah I think ill fix the code (y). the spaces are too big.